tests/js/spec/components/group/sidebar.spec.jsx

```diff
@@ -114,37 +133,54 @@ describe('GroupSidebar', function () {
         body: [],
       });
 
-      wrapper = mountWithTheme(
-        <GroupSidebar
-          api={new MockApiClient()}
-          group={group}
-          organization={organization}
-          project={project}
-          event={TestStubs.Event()}
-          environments={[environment]}
-        />,
+      const wrapper = mountWithTheme(
+        <CommittersProvider>
+          <GroupSidebar
+            api={new MockApiClient()}
+            group={group}
+            organization={organization}
+            project={project}
+            event={TestStubs.Event()}
+            environments={[environment]}
+          />
+        </CommittersProvider>,
         routerContext
       );
+
       await tick();
       wrapper.update();
-    });
 
-    it('renders no tags', function () {
       expect(wrapper.find('GroupTagDistributionMeter')).toHaveLength(0);
-    });
-
-    it('renders empty text', function () {
       expect(wrapper.find('[data-test-id="no-tags"]').text()).toBe(
         'No tags found in the selected environments'
       );
     });
   });
 
   describe('environment toggle', function () {
-    it('re-requests tags with correct environment', function () {
+    it('re-requests tags with correct environment', async function () {
+      const tagsMock = MockApiClient.addMockResponse({
+        url: '/issues/1/tags/',
+        body: TestStubs.Tags(),
+      });
+
+      const wrapper = mountWithTheme(
+        <GroupSidebar
+          api={new MockApiClient()}
+          group={TestStubs.Group()}
+          organization={organization}
+          project={project}
+          event={TestStubs.Event()}
+          environments={[environment]}
+        />,
+        {routerContext, Providers: CommittersProvider}
```

Comment:
can't we just wrap the component with the provider?

Reply:
No, because we are calling setProps and that can only be called on the root...